plugin.xml

```diff
@@ -35,8 +37,8 @@
         <framework src="com.adobe.creativesdk.foundation:auth:0.9.1062+"/>
         <source-file src="src/android/AdobeAuthCredentialsApp.java" target-dir="src/com/adobe/phonegap/csdk" />
         <config-file target="res/values/strings.xml" parent="/resources">
-            <string name="creative_sdk_client_id">$CREATIVE_SDK_CLIENT_ID</string>
-            <string name="creative_sdk_client_secret">$CREATIVE_SDK_CLIENT_SECRET</string>
+            <string name="csdk_client_id_android">$CSDK_CLIENT_ID_ANDROID</string>
```

Comment:
This is fine but you don't need to change `name="creative_sdk_client_id"` to `name="csdk_client_id_android"` but that's cool.